### PR TITLE
[GUI] Fallback to NPM when Yarn isn't installed

### PR DESCRIPTION
**Pull request by jamiedavenport**
Closes: #606 

### What are the changes and their implications?

Uses `hasbin` to determine if Yarn is installed and uses that as the default. Otherwise, it falls back to using NPM.

### Checklist

- [ ] Tests added for changes
- [ ] User facing changes documented

### CODE REVIEW

File: packages/gui/package.json [hunk 1]
```diff
@@ -13,7 +13,8 @@
     "heroicons-react": "^1.0.15",
     "react": "experimental",
     "react-dom": "experimental",
-    "typescript": "3.9.3"
+    "typescript": "3.9.3",
+    "hasbin": "1.2.3"
```
Comment (merelinguist) [inline packages/gui/package.json:17]: Nice! It’s a tiny nitpick, but can you move hasbin up so it’s all alphabetical?

Comment (jamiedavenport) [reply packages/gui/package.json]: Sorted

**Comment by merelinguist** [comment]
Merge when ready.

**Comment by merelinguist** [comment]
@all-contributors please add @jamiedavenport for code

**Pull request merged**